src/main/java/com/epam/aidial/core/config/FileConfigStore.java

```diff
@@ -127,7 +127,7 @@ private static InputStream openStream(String path) {
         }
     }
 
-    private static void setMissingFeatures(Deployment model, Features features) {
+    private static void inheritMissingFeatures(Deployment model, Features features) {
```

Comment:
The name of the first argument confuses a bit: there is a base class but the name is model. Moreover it's always Assistant.

Reply:
Renamed back to `setMissingFeatures` to avoid confusion